### PR TITLE
docs: add list of aio example apps

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

This is a work in progress, but the end result should be a list of all of our StackBlitz examples in one place that visitors could browse and go directly to the examples without going to the guides.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/example-apps-list.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+## Angular CLI builders
+
+<live-example name="cli-builder"></live-example>
+
+Demonstrates using Angular CLI builders.
+For more information, see [Angular CLI builders](guide/cli-builder) and the associated [GitHub repository](https://github.com/mgechev/cli-builders-demo).
+
+## Comparing observables to other techniques
+
+<live-example name="comparing-observables"></live-example>
```
Comment (gkalpak) [inline aio/content/guide/example-apps-list.md:84]: Drive-by comment: Keep in mind that some examples are not polished enough to be shown as standalone StackBlitz projects. (They are only intended to be used for extracting doc-regions for code snippets.

Also, not all examples can be opened on StackBlitz (i.e. not all of them have a `stackblitz.json` file).

**Review by aikithoughts — COMMENTED** [review]
This a great initial commit! 

Reviewed-for: global-docs-approvers

**Review by aikithoughts — APPROVED** [review]
LGTM

Reviewed-for: global-docs-approvers

**Review by IgorMinar — APPROVED** [review]
LGTM. Thanks!

Reviewed-for: code-ownership

File: aio/content/guide/example-apps-list.md [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+## Fundamentals
+
+These examples demonstrate minimal, fundamental concepts.
+
+### Getting started application
+
+<live-example name="getting-started"></live-example>
+
+Introductory application demonstrating Angular features.
+For more information, see [Getting started](start/).
```
Comment (gkalpak) [inline aio/content/guide/example-apps-list.md:14]: Nit: The trailing `/` is redundant:

```suggestion
For more information, see [Getting started](start).
```

File: aio/content/guide/example-apps-list.md [hunk 3]
```diff
@@ -0,0 +1,476 @@
+# Example applications
+
+The following is a list of the example applications in the [Angular documentation](https://angular.io/docs).
```
Comment (gkalpak) [inline aio/content/guide/example-apps-list.md:3]:
```suggestion
The following is a list of the example applications in the [Angular documentation](docs).
```

(By including the `https`, it makes it (incorrectly) appear as an external link (with the "external link" icon next to it).)

**Review by gkalpak — APPROVED** [review]
Nice!
